tests/ci/submit_AzureML_pytest.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
there are several files very similar, `submit_AzureML_pytest.py` and `submitpytest.py`. 

Also, in the rest of the code we are following the pattern of using lowercase for filenames `submit_azureml_pytest.py`

Reply:
changed name and removed old.